backend/services/tasks/pokemon_tasks.py

```diff
@@ -0,0 +1,34 @@
+from celery.utils.log import get_task_logger
+
+from common.constants import POKEAPI_POKEMONS_LIMIT
+from pokebattle.celery import app as celery_app
+from pokemons.services import get_pokemon_list
+from pokemons.models import Pokemon
+
+logger = get_task_logger(__name__)
+
+
+@celery_app.task
+def save_all_pokemons_from_pokeapi():
+    pokemons = get_pokemon_list(POKEAPI_POKEMONS_LIMIT)
```

Comment:
Fetching all 1000 pokemons at once might work alright for this particular case but in real world situations you will be dealing with much bigger data sizes and you won't be able to fetch everything at once. Ideally you paginate your requests into reasonable chunk sizes and fetch things in multiple requests.

Reply:
Also, real work APIs will have a hard limit on how many items you can fetch at once. So there's no other way to fetch everything but to paginate requests

Reply:
Got it!